docs/modules/math.md

```diff
@@ -113,9 +113,13 @@ Parameter  | Description   |  Required?
 Return the smallest integer greater than or equal to a specified number.
 
 ``` 
-Math.ceil(number)
+Math.ceil(number, digits)
 ```
 
+Parameter  | Description   |  Required?
+---------- | ------------- | ---------:
+`digits`   | A number indicating the number of significant digits to be used. When `digits` is 0, the number is rounded to the nearest integer;  `digits` greater than 0, then number is rounded to the specified number of decimal places; `digits` less than  0, the number is rounded to the left of the decimal point. | No
+
```

Comment:
Let's reword this a bit, how about:

The number of significant digits to be used. If 0, the value is rounded to the nearest integer; if greater than 0, the value is rounded to the specified number of decimal places; if less than 0, the value is rounded to the left of the decimal point.